modules/sdk/src/main/scala/org/tessellation/sdk/domain/statechannel/StateChannelValidator.scala

```diff
@@ -40,12 +44,26 @@ object StateChannelValidator {
       signedValidator
         .isSignedExclusivelyBy(signedSC, address)
         .map(_.errorMap[StateChannelValidationError](_ => NotSignedExclusivelyByStateChannelOwner))
+
+    private def validateSnapshotSize(
+      signedSC: Signed[StateChannelSnapshotBinary]
+    ): F[StateChannelValidationErrorOr[Signed[StateChannelSnapshotBinary]]] =
+      signedSC.toBinaryF.map { binary =>
```

Comment:
I find it odd that we're calculating the size of the binary with the signatures. Specifically that the size of the signatures may vary. But if that's the requirement than ok.

Reply:
well, actually it should take into account signatures 🤔 shouldn't it @TheMMaciek?

Reply:
Yeah that is a requirement to take signatures into account, to not allow flooding with signatures in the future.